spec/fixtures/module/run-as-node.js

```diff
@@ -3,3 +3,4 @@ console.log(JSON.stringify({
   processType: typeof process.type,
   window: typeof window
 }))
+process.exit(0)
```

Comment:
Can you create a separate fixture for this spec, the other specs using this file might not handle process exit.